### ISSUE TITLE
Please add this to nuget package manager

### ISSUE THREAD

**Issue by Shadowblitz16** [DevJohnC/OpenAL.NET#1]
Please add this to nuget package manager for visual studio 2017.
preferably for he same .net runtimes as opengl.

**Comment by DevJohnC** [comment]
The library hasn't been worked on for quite a time. I'll need to take some time to update it a touch to work on 64bit builds and get the OpenAL dll updated. Then I should be able to make a nuget package for it.

**Comment by Shadowblitz16** [comment]
ok cool I was just wondering since I might try making a game in opengl soon.
does this support ogg vorbis and loop tags?

**Comment by DevJohnC** [comment]
I don't think this library will do what you're looking for - there are no codecs or advanced audio features here (besides positional audio I suppose). OpenAL (and thus these bindings) are for playing raw audio samples, that's raw PCM.

**Comment by Shadowblitz16** [comment]
so this library would be good for creating my own sound and music editor then?

**Comment by krestenlaust** [comment]
Yes, I believe so
